### PR TITLE
Add zigbee-switch for Vimar xx593

### PR DESCRIPTION
**Pull request by maxelweb-vimar**
Add support with a custom sub-driver for:

- Vimar Smart Actuator with Power Metering (xx593)

### CODE REVIEW

File: drivers/SmartThings/zigbee-switch/src/zigbee-switch-power/vimar/init.lua
Comment (greens) [inline drivers/SmartThings/zigbee-switch/src/zigbee-switch-power/vimar/init.lua:38]: this will be sent by `device:configure` already

I would suggest writing a test to verify that you're seeing all the messages you'd expect from configure

**Review by greens — COMMENTED** [review]
no need for a new profile

firmwareupdate is managed by the hub

**Comment by maxelweb-vimar** [comment]
Hi @lelandblue and @greens, I have seen your replies on #202. Thanks for the merge. 😃 

Can you also proceed with the merge of this branch since the certification of this device has passed?

**Review by lelandblue — APPROVED** [review]

**Comment by greens** [comment]
@maxelweb-vimar Hi. Unfortunately it looks like we've introduced some conflicts upstream. Do you think you could rebase your branch and resolve them?

**Comment by maxelweb-vimar** [comment]
Hi @greens, I think this is normal since we merged PR #202.
I've resolved all the conflicts with upstream, as requested. Now this branch is up-to-date. 

Waiting for your approval, thanks in advance 👍🏼

**Review by greens — APPROVED** [review]

**Pull request merged**
